bin/cli.js

```diff
@@ -18,7 +18,7 @@ const cliOptions = [
   [ '--node-only <patterns>',    'Node only tests patterns', splitOpts, [] ],
   [ '--exclude <patterns>',      'Exclude tests patterns', splitOpts, [] ],
   [ '--browsers <values>',       'Browsers to run', splitOpts, [] ],
-  [ '-l, --browser-log <value>', 'Browser log level' ],
+  [ '-l, --log-level <value>',   'Log level' ],
```

Comment:
I'd prefer to have verboseness by counting `-v` options passed to program as we did here https://github.com/metarhia/jstp/pull/366/files#diff-22d5d2b57c69d2e1ac054c4d0f744ff1R13.
i.e. `-v` - quiet (errors only) - default
     `-vv` -  warn
etc.

'disable' level makes no sense as you will still print errors even with disable so it automatically becomes 'error' level.

Reply:
In this case, I think that general verboseness and log level for karma should be separated, because node will basically have just `error` or `log`. And since show option is needed only for config, it'll be better to leave `--browser-log-level` as it was and add single option `--show-config`. Wdyt?

Reply:
I think it'll be okay to have 4 levels

* 'error' - default
* 'warn' - `-v`
* 'info' - `-vv`
* 'debug'  - `-vvv`

Somewhat aligned with karma (we ignore disabled as it seems strange in this context). Even though we will only use 'error' and 'info' (config should be printed starting with 'info') for now.
wdyt?

Reply:
I think that end user can misunderstand this option because logs from `metatests` will be printed anyway. This means that only thing that verboseness level changes is whether config will be printed or not when all other levels will be used only for browser. Also, I'd like to have consistent option names across both config file and command line options